### PR TITLE
fix(ivy): WorkerAppModule should be an APP_ROOT

### PR DESCRIPTION
**Pull request by marclaval**
Fixes FW-1021

### CODE REVIEW

**Comment by marclaval** [comment]
```javascript
export const ROOT_CONTEXT = new InjectionToken<RootContext>(
    'ROOT_CONTEXT_TOKEN',
    {providedIn: 'root', factory: () => createRootContext(inject(SCHEDULER))});
```

Without this fix, this token cannot be found in a web worker scenario because these is no actual root injector (as defined by `APP_ROOT`).

File: modules/playground/e2e_test/web_workers/kitchen_sink/kitchen_sink_spec.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import {ExpectedConditions, browser, by, element, protractor} from 'protractor';
 
 import {verifyNoBrowserErrors} from '../../../../e2e_util/e2e_util';
 
-fixmeIvy('NullInjectorError: No provider for InjectionToken ROOT_CONTEXT_TOKEN!')
+fixmeIvy('FW-1032: FW-1032: Ivy doesn\'t support injecting Renderer, the deprecated renderer V1')
```
Comment (alfaproject) [inline modules/playground/e2e_test/web_workers/kitchen_sink/kitchen_sink_spec.ts]: JIRA ticket is duplicated

**Comment by alfaproject** [comment]
You forgot to remove the `fixmeIvy` import in some files

**Review by mhevery — APPROVED** [review]

**Comment by kara** [comment]
[presubmit](http://test/OCL:232698366:BASE:232698425:1549477431755:5bc6706b)

**Comment by kara** [comment]
merge-assistance: Misko giving global approval

**Pull request closed without merging**